Apps/phdossier/dossier.json

```diff
@@ -0,0 +1,3522 @@
+{
+    "appid": "341d81e0-5308-4d4d-93dd-d20a2a01d941",
+    "name": "Dossier",
+    "description": "This app supports investigate actions on the Infoblox Dossier platform",
+    "type": "information",
+    "product_vendor": "Infoblox",
+    "logo": "logo_dossier.svg",
+    "logo_dark": "logo_dossier_dark.svg",
+    "product_name": "Dossier",
+    "product_version_regex": ".*",
+    "publisher": "Nick Roy",
+    "license": "Copyright (c) Nick Roy, 2019",
```

Comment:
These two lines should be Splunk (Nick is one of us).

Reply:
Fixed.